packages/discord.js/src/structures/GuildChannel.js

```diff
@@ -327,11 +327,16 @@ class GuildChannel extends BaseChannel {
    * @returns {Promise<GuildChannel>}
    * @example
    * // Add a parent to a channel
-   * message.channel.setParent('355908108431917066', { lockPermissions: false })
-   *   .then(channel => console.log(`New parent of ${message.channel.name}: ${channel.name}`))
+   * message.channel.setParent('355908108431917066')
+   *   .then(channel => console.log(`New parent of ${channel.name}: ${channel.parent.name}`))
+   *   .catch(console.error);
+   * @example
+   * // Move a channel and sync its permissions with the parent
+   * message.channel.setParent('355908108431917066', { lockPermissions: true })
+   *   .then(channel => console.log(`Moved ${message.channel.name} to ${channel.parent.name}`))
    *   .catch(console.error);
    */
-  setParent(channel, { lockPermissions = true, reason } = {}) {
+  setParent(channel, { lockPermissions = false, reason } = {}) {
```

Comment:
What's Discord's default behaviour? Do we need this default value at all?

Reply:
The default API behavior is what this PR is changing. Setting lockPermissions to true changes the permissionOverwrites as calculated by djs, setting it to false only changes the parent and nothing else. On the client, it usually prompts the user whether they want to sync or not.
In summary: false is the default behavior